### ISSUE TITLE
TTC code example

### ISSUE THREAD

**Issue by viktorengstrom** [mal-lang/mal-documentation#2]
All symbols have a corresponding code example except for TTC. Should it have one and, if so, what?

**Comment by mathiasekstedt** [comment]
Yes we should have a code example of TTC. Actually two. Both a keyword (e.g. HardAndUncertain) and a mathematical distribution (e.g. exponential(3))
